pkg/controllers/gracefuleviction/crb_graceful_eviction_controller.go

```diff
@@ -68,9 +68,8 @@ func (c *CRBGracefulEvictionController) syncBinding(binding *workv1alpha2.Cluste
 	}
 
 	objPatch := client.MergeFrom(binding)
-	modifiedObj := binding.DeepCopy()
-	modifiedObj.Spec.GracefulEvictionTasks = keptTask
-	err := c.Client.Patch(context.TODO(), modifiedObj, objPatch)
+	binding.Spec.GracefulEvictionTasks = keptTask
```

Comment:
/cc @XiShanYongYe-Chang for verifying there is extra usage.

Reply:
The situation here requires Deepcopy, refer to here:
vendor/sigs.k8s.io/controller-runtime/pkg/controller/controllerutil/controllerutil.go:250

Reply:
Thanks for reminding.